### PR TITLE
[WIP] Use YouTube for storage

### PR DESCRIPTION
**Pull request by ivantjh**

### CODE REVIEW

**Comment by ivantjh** [comment]
Do not merge this first

**Comment by duncanleo** [comment]
screen shot 2017-12-11 at 9 22 08 pm

**Comment by duncanleo** [comment]
screen shot 2017-12-11 at 10 05 21 pm

**Comment by duncanleo** [comment]
4/zm_Cd-5o1e7SRzF5Pxo19drSC_kUEw-l76V3b9rlGDg

File: app/lib/yt_auth.rb [hunk 1]
```diff
@@ -59,7 +59,11 @@ def user_credentials_for(scope)
       url = authorizer.get_authorization_url(base_url: OOB_URI)
       puts "Open the following URL in your browser and authorize the application."
       puts url
-      code = ENV['GOOGLE_OAUTH_RESPONSE_CODE']
+
+      puts "You have 20s to enter the response code in #{ENV['GOOGLE_OAUTH_RESPONSE_CODE']}"
+      sleep(20)
```
Comment (ivantjh) [inline app/lib/yt_auth.rb]: wtf

Comment (duncanleo) [reply app/lib/yt_auth.rb]: doesn't work anyway

**Comment by duncanleo** [comment]
Work abandoned in favour of an S3 + Elastic Transcoder setup

**Pull request closed without merging**